### PR TITLE
ST-329: Content update for Pension and Employment Success

### PR DESCRIPTION
**Pull request by SeanArmstrong**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [x] Reviewed
- [x] Welsh

### CODE REVIEW

**Comment by SeanArmstrong** [comment]
https://github.com/hmrc/tai-acceptance-test-suite/pull/103

File: app/controllers/income/estimatedPay/update/IncomeUpdateCalculatorController.scala [hunk 1]
```diff
@@ -89,17 +89,23 @@ trait IncomeUpdateCalculatorController extends TaiBaseController
...
 
         employmentService.employment(Nino(user.getNino), id) flatMap {
-          case Some(employment) =>
+          case Some(employment: Employment) =>
+
+            val incomeType = incomeTypeIdentifier(employment.receivingOccupationalPension)
+
             for {
-              incomeToEdit <- incomeService.employmentAmount(Nino(user.getNino), id)
+              incomeToEdit: EmploymentAmount <- incomeService.employmentAmount(Nino(user.getNino), id)
```
Comment (lewey) [inline app/controllers/income/estimatedPay/update/IncomeUpdateCalculatorController.scala]: Can these be ran concurrently?

File: test/controllers/IncomeControllerSpec.scala [hunk 2]
```diff
@@ -315,41 +317,62 @@ class IncomeControllerSpec extends PlaySpec
 
   "updateEstimatedIncome" must {
     "return OK" when {
-      "income is successfully updated" in {
+
+      def expectedSuccessPage(employerName: String, employerId: Int,
```
Comment (lewey) [inline test/controllers/IncomeControllerSpec.scala]: Is this being used?

Comment (SeanArmstrong) [reply test/controllers/IncomeControllerSpec.scala]: Nope it was old from when i was trying to run the specs a different way

File: app/controllers/income/estimatedPay/update/IncomeUpdateCalculatorController.scala (hunk 1 above)
Comment (SeanArmstrong) [reply app/controllers/income/estimatedPay/update/IncomeUpdateCalculatorController.scala]: They can. All I added was a type though :(

Comment (lewey) [reply app/controllers/income/estimatedPay/update/IncomeUpdateCalculatorController.scala]: Sorry chap. This was meant to be a 'as you are in the area how about this?'. Perhaps there is a different way to flag these.

**Pull request merged**